addons/parachute/functions/fnc_doLanding.sqf

```diff
@@ -18,6 +18,7 @@ private ["_unit"];
 _unit = _this select 0;
 GVAR(PFH) = false;
 [_unit, "AmovPercMevaSrasWrflDf_AmovPknlMstpSrasWrflDnon", 2] call EFUNC(common,doAnimation);
+SETVAR(_unit,chuteIsCut,false);
```

Comment:
Should be `SETGVAR`, so the variable name gets correctly prefixed, but that macro doesn't exist. God, this macros are totally unnecesary. Please use this instead:

`_unit setVariable [QGVAR(chuteIsCut), false]`